OrbitGl/PickingManagerTest.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TESTS_PICKING_MANAGER_TEST_H_
+#define ORBIT_GL_TESTS_PICKING_MANAGER_TEST_H_
+
+#include "GlCanvas.h"
+#include "PickingManager.h"
+#include "absl/base/casts.h"
+
+class PickableMock : public Pickable {
+ public:
+  void OnPick(int, int) override { picked_ = true; }
```

Comment:
here we need context add /* */ with parameter names.

Reply:
Done